src/wp-includes/js/customize-preview-widgets.js

```diff
@@ -223,9 +223,9 @@ wp.customize.widgetsPreview = wp.customize.WidgetCustomizerPreview = (function(
 		 *
 		 * @since 4.5.0
 		 *
-		 * @returns {Array.<{before: Comment, after: Comment, instanceNumber: number}>|void} Either an array with an object
+		 * @returns {Array.<{before: Comment, after: Comment, instanceNumber: number}>} Either an array with an object
```

Comment:
Is this format in line with the documentation standards? Feels like there's too much data in the return type.

Additionally,  usage of `Either` needs to be accompanied by `or`.

Reply:
According to @herregroen it's in line with the standards. [The documentation standards](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/) are unclear about this.